recipes/libudev-devel-cos6-x86_64/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr
+pushd ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr > /dev/null 2>&1
+cp -Rf "${SRC_DIR}"/binary/* .
+
+# For some reason /usr/lib64/libudev.so is relative linked to /lib64/libudev.so.0.5.1
+# and not /usr/lib64/libudev.so.0.5.1
+# since our sysroot doesn't have /lib64, we must remake the link ourselves
```

Comment:
> sysroot doesn't have /lib64

It does, if you use `sysroot` instead of `sysroot/usr` in both recipes, see above.

Reply:
Alright, devel package should be unpacked to /usr, non devel should be unpacked /.

Reply:
No, both to the same sysroot, meaning `/` or, for the recipe, `${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/`

Reply:
Oh, unless `conda-build` is being "clever" and removes the leading directory if it is only top-level thing..

Reply:
Might very well be:
```sh
curl -sL http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libudev-devel-147-2.73.el6_8.2.x86_64.rpm | rpm2cpio | cpio -t
./usr/include/libudev.h
./usr/lib64/libudev.so
./usr/lib64/pkgconfig/libudev.pc
./usr/share/doc/libudev-devel-147
./usr/share/doc/libudev-devel-147/COPYING
./usr/share/gtk-doc/html/libudev/api-index-full.html
./usr/share/gtk-doc/html/libudev/ch01.html
./usr/share/gtk-doc/html/libudev/home.png
./usr/share/gtk-doc/html/libudev/index.html
./usr/share/gtk-doc/html/libudev/index.sgml
./usr/share/gtk-doc/html/libudev/left.png
./usr/share/gtk-doc/html/libudev/libudev-udev-device.html
./usr/share/gtk-doc/html/libudev/libudev-udev-enumerate.html
./usr/share/gtk-doc/html/libudev/libudev-udev-list.html
./usr/share/gtk-doc/html/libudev/libudev-udev-monitor.html
./usr/share/gtk-doc/html/libudev/libudev-udev-queue.html
./usr/share/gtk-doc/html/libudev/libudev-udev.html
./usr/share/gtk-doc/html/libudev/libudev.devhelp
./usr/share/gtk-doc/html/libudev/libudev.devhelp2
./usr/share/gtk-doc/html/libudev/right.png
./usr/share/gtk-doc/html/libudev/style.css
./usr/share/gtk-doc/html/libudev/up.png
./usr/share/pkgconfig/udev.pc
373 blocks
```
=> only `usr` in the top level.

Reply:
Right, and non devel has two things

```
(base) mark@mark-xps:~/git/feedstocks/staged-recipes/recipes/libudev-cos6-x86_64$ curl -sL http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libudev-147-2.73.el6_8.2.x86_64.rpm | rpm2cpio | cpio -t
./lib64/libudev.so.0
./lib64/libudev.so.0.5.1
./usr/share/doc/libudev-147
./usr/share/doc/libudev-147/COPYING
```

Reply:
Mystery quickly solved, I guess :D. I didn't even know you could specify deeper paths with `folder: binary/usr`, nice! Much cleaner workaround than in `build.sh`. And even with an explanatory comment -- perfect!
Thanks for opening https://github.com/conda/conda-build/issues/3595!